src/main/scala/no/ndla/draftapi/model/api/NDLAErrors.scala

```diff
@@ -51,6 +51,7 @@ object Error {
 case class NotFoundException(message: String, supportedLanguages: Seq[String] = Seq.empty)
     extends RuntimeException(message)
 case class ArticlePublishException(message: String) extends RuntimeException(message)
+case class ValidationException(message: String = Error.VALIDATION_DESCRIPTION) extends RuntimeException(message)
```

Comment:
Kanskje legge til feilhåntering for denne i `NdlaController` sånn at bruker får tilbake en ikke generisk feilmelding?

Reply:
Aye. Will fix

Reply:
Ah, ser ut som at det allerede finnes en `ValidationException` i `no.ndla.validation`. Bruker heller den. Den er konfigurert i `NdlaController` også

Reply:
sounds good :+1: